.github/workflows/lint_and_suggest_pr.yml

```
@@ -13,3 +13,13 @@ jobs:
         with:
           mode: suggest
           lint-bib-file: true
+      - name: All Mathlib files have lean or md extension
```

(Perhaps it's best to quote this when adding a dot, out of paranoia.)
```suggestion
      - name: All Mathlib files have a .lean or .md extension